src/geom_handlers/common_assessment.jl

```diff
@@ -0,0 +1,235 @@
+""" Functions common to both site_assessment methods. """
```

Comment:
Remove errant whitespace

```suggestion
""" Functions common to both site_assessment methods."""
```